core/src/main/java/tc/oc/pgm/rotation/vote/VotePoolOptions.java

```diff
@@ -3,14 +3,16 @@
 import com.google.common.collect.Sets;
 import java.util.Map;
 import java.util.Set;
+import java.util.UUID;
 import java.util.stream.Collectors;
+import org.jetbrains.annotations.Nullable;
 import tc.oc.pgm.api.map.MapInfo;
 import tc.oc.pgm.rotation.pools.VotingPool;
 
 public class VotePoolOptions {
 
-  // Set of maps to be used in custom vote selection
-  private final Set<MapInfo> customVoteMaps;
+  // Set of maps w/ related data to be used in custom vote selection
+  private final Set<CustomMapEntry> customVoteMaps;
```

Comment:
I don't like this one, because it no longer guarantees there aren't any duplicates, potentially change to Set<MapInfo, UUID> instead